hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/utils/PrecompilePricingUtils.java

```diff
@@ -142,7 +147,7 @@ public long gasFeeInTinybars(
                         .setSignedTransactionBytes(signedTxn.toByteString())
                         .build();
 
-        final var accessor = SignedTxnAccessor.uncheckedFrom(txn);
+        final var accessor = accessorFactory.constructSpecializedAccessor(txn.toByteArray());
```

Comment:
May be we could add `uncheckedSpecializedAccessor` that catches `Exception` in `AccessorFactory` and throws `IllegalArgumentException` similar to `SignedTxnAccessor.uncheckedForm`

```
    public SignedTxnAccessor uncheckedSpecializedAccessor(byte[] signedTxnWrapperBytes) {
        try {
            return constructSpecializedAccessor(signedTxnWrapperBytes);
        } catch (InvalidProtocolBufferException e) {
            throw new IllegalArgumentException("Not a valid signed transaction");
        }
    }